bin/lesgo-scripts.sh

```diff
@@ -94,6 +94,13 @@ NC='\033[0m';
 #                                                                             #
 ###############################################################################
 
+SERVERLESS_VERSION=`npm view serverless version`
+echo -e "Running Serverless version: ${SERVERLESS_VERSION}"
+
+SERVERLESS_VERSION_NUMBER=$((${SERVERLESS_VERSION:0:1}  + 0))
```

Comment:
@ELortiz This won't work whenever we reach 2 digits version. but this is fine for now. As most likely at that time the commands might eventually change anyways. But just take note

Reply:
@xXLXx  yeah makes sense updated now the code. Thanks